### DISCUSSION TITLE
Editing and deployment of Edge Functions from within the Supabase Dashboard

### DISCUSSION THREAD

**Discussion by stevenjwheeler** [supabase/supabase#13749] [Feature Requests]
Instead of using the terminal to write my functions and then deploy them, I would like to also be able to go into the Supabase Dashboard and open up a coding window where I can code my function and then deploy it by just pressing a button, making it easier to write functions from any device and easily deploy them.

We already have coding windows such as the GraphiQL window in API docs, so it could be reused for this feature.

**Comment by organicnz** [comment]
Great suggestion :)
